ReleaseProcessAutomation/Git/CommandLineGitClient.cs

```diff
@@ -299,9 +299,10 @@ public void CheckoutDiscard (string fileName)
     ExecuteGitCommand($"checkout -- {fileName}");
   }
 
-  public void Tag (string arguments)
+  public void Tag (string tagName)
```

Comment:
As discussed, a better soluition would be to pass both the tagname and the message as separate parameters. The GitClient should only do the commandline operation, not define (part of) the actual message.